app/models/discussion.rb

```diff
@@ -8,12 +8,16 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
+  after_create :find_or_create_subscriptions
   after_touch :notify_new_comment
 
   private
 
+  def find_or_create_subscriptions
+    Subscriber.for_discussion(self).subscribe(User.all)
+  end
+
   def notify_new_comment
-    subscriptions = Subscriber.for_discussion(self).subscribe(User.all)
-    subscriptions.map(&:new_comment)
+    find_or_create_subscriptions.map(&:new_comment)
   end
 end
```

Comment:
Yeah, I found it a bit odd to create subscriptions on something called `notify_new_comment`.

This still has two issues, imo, which we might not want to address yet (or ever):

1. I am a new user on the platform, which has been running for a while. For every comment on an existing discussion I'll get subscribed to it.

2. I have no way of unsubscribing to a discussion. Even if I delete the subscription, it will get created again.